secure.go

```diff
@@ -68,6 +71,8 @@ type Secure struct {
 	badHostHandler http.Handler
 }
 
+var defaultHeader = make(http.Header)
```

Comment:
I think this is in here by mistake. It's not referenced anywhere else -- I remember I was seeing if I could optimize adding header values but I must not have removed this.